its/plugin/tests/src/test/java/com/sonar/it/php/Tests.java

```diff
@@ -132,33 +125,12 @@ static Double getMeasureAsDouble(String componentKey, String metricKey) {
   @CheckForNull
   static Components.Component getComponent(String projectKey, String componentKey) {
     List<Components.Component> components = newWsClient().components().tree(new TreeRequest()
-      .setComponent(projectKey)
-      .setQ(componentKey))
+        .setComponent(projectKey)
+        .setQ(componentKey))
       .getComponentsList();
     return components.size() == 1 ? components.get(0) : null;
   }
 
-  /**
-   * Extract analysis warnings from component task to evaluate if expected warnings are send to the server
-   */
-  static List<String> getAnalysisWarnings(BuildResult result) {
-    String taskId = getTaskId(result);
-    if (taskId == null) {
-      throw new RuntimeException("Task id can not be processed from BuildResult");
-    }
-    CeService service = newWsClient().ce();
-    return service.task(TASK_REQUEST.setId(taskId)).getTask().getWarningsList();
-  }
```

Comment:
There is a difference between AnalysisWarnings and the WARN records in the log. AnalysisWarnings are UI warnings. You should not remove the method and the assertions used to test AnalysisWarnings

Reply:
Nice to know